examples/webhook.php

```diff
@@ -58,11 +58,11 @@ public function processWebhook()
 
         // verify hmac256
         $headers = getallheaders();
-        $sig = $headers['Btcpay-Sig'];
+        $sig = $headers['BTCPay-Sig'];
```

Comment:
This can be a tricky one, according to RFC there is no special case convention here. So Nginx with mod PHP behaves differently than FPM, or Apache. I did not find a better solution yet but to lowercase everything and only then check the header if it exists like that:
```
// Validate webhook request.
// Note: getallheaders() CamelCases all headers for PHP-FPM/Nginx but for others maybe not, so "BTCPay-Sig" may becomes "Btcpay-Sig".
$headers = getallheaders();
foreach ($headers as $key => $value) {
	if (strtolower($key) === 'btcpay-sig') {
		$signature = $value;
	}
}
```